content/docs/1.13/deployment.md

```diff
@@ -389,13 +389,6 @@ Production deployments need an external process which aggregates data and create
 Project [spark-dependencies](https://github.com/jaegertracing/spark-dependencies) is a Spark job which derives
 dependency links and stores them directly to the storage.
 
-## Configuration
```

Comment:
Why removing this? Is it documented somewhere else?

Reply:
Since we have a full CLI subpage, I felt like this wasn't saying much that's not said on CLI docs.

Reply:
I  restored the section and linked to CLI docs.